.github/workflows/sync-nest-data.yaml

```
@@ -11,6 +11,7 @@ env:
 jobs:
   update-data:
     name: Run data sync
+    if : github.repository == 'OWASP/Nest' && github.ref == 'refs/heads/main'
```

This should be enough
```suggestion
    if: github.repository == 'OWASP/Nest'
```